public/app/plugins/datasource/influxdb/components/VisualInfluxQLEditor/Seg.tsx

```diff
@@ -106,6 +106,7 @@ const SelSingleLoad = ({ loadOptions, allowCustomValue, onChange, onClose }: Sel
   return (
     <div className={selectClass}>
       <Select
+        noOptionsMessage={loadState.loading ? 'Loading options...' : 'No options found'}
```

Comment:
This should be less hacky and give a nice spinner for free (BaseSelect supports it):

```suggestion
        isLoading={loadState.loading}
```

Reply:
wow, good catch! i should have looked at the props.. i guess i assumed a "not async" select-box will not have an isLoading, but here it is! 😄